time_tracker_api/time_entries/time_entries_repository.py

```diff
@@ -177,57 +142,38 @@ def add_complementary_info(
     def find_all(
         self,
         conditions,
-        custom_sql_conditions,
         event_context: EventContext,
         date_range: dict = None,
+        owner_ids: list = None,
         test_user_ids=None,
         offset=0,
         max_count=None,
         visible_only=True,
-        custom_params: dict = None,
         mapper: Callable = None,
     ):
-        partition_key_value = self.find_partition_key_value(event_context)
         max_count = self.get_page_size_or(max_count)
+        date_range = date_range if date_range else {}
 
-        params = [
-            {"name": "@partition_key_value", "value": partition_key_value},
-            {"name": "@offset", "value": offset},
-            {"name": "@max_count", "value": max_count},
-        ]
-
-        params.extend(self.generate_params(conditions))
-        params.extend(custom_params)
-
-        query_str = """
-            SELECT * FROM c
-            WHERE c.{partition_key_attribute}=@partition_key_value
-            {conditions_clause}
-            {visibility_condition}
-            {test_users_exclusion_condition}
-            {custom_sql_conditions_clause}
-            {order_clause}
-            OFFSET @offset LIMIT @max_count
-            """.format(
-            partition_key_attribute=self.partition_key_attribute,
-            visibility_condition=self.create_sql_condition_for_visibility(
-                visible_only
-            ),
-            conditions_clause=self.create_sql_where_conditions(conditions),
-            test_users_exclusion_condition=self.create_sql_test_users_exclusion_condition(
-                test_user_ids
-            ),
-            custom_sql_conditions_clause=self.create_custom_sql_conditions(
-                custom_sql_conditions
-            ),
-            order_clause=self.create_sql_order_clause(),
+        query_builder = (
+            CosmosDBQueryBuilder()
+            .add_sql_in_condition('owner_id', owner_ids)
+            .add_sql_where_equal_condition(conditions)
+            .add_sql_visibility_condition(visible_only)
+            .add_date_range(date_range)
```

Comment:
I consider. the method should be named `.add_sql_in_date_range_condition` for consistency.

Reply:
Same here. I consider TimeEntriesquerybuilder should be used.

Reply:
Ok, adjustments were done.